Tasks/NuGetInstaller/NuGetInstaller.ps1

```diff
@@ -1,5 +1,6 @@
 param(
```

Comment:
There's a lot of new code here. Consider sprinkling some Write-Verbose statements where appropriate.